kie-wb-parent/kie-wb-webapp/src/main/resources/security-module-process-admin.properties

```diff
@@ -19,6 +19,7 @@ role.process-admin.permission.perspective.read.UserHomePagePerspective=true
 role.process-admin.permission.perspective.read.ProcessDefinitions=true
 role.process-admin.permission.perspective.read.ProcessInstances=true
 role.process-admin.permission.perspective.read.Tasks=true
+role.process-admin.permission.perspective.read.TaskAdmin=true
```

Comment:
I'm running the same, `kie-wb-webapp.war`. I build this by running `mvn clean install` with your PR checked out in kie-wb-common, jbpm-wb and then kie-wb-distributions.

It is strange, that you can't reproduce it, because the visibility of Task Administration in the menu is consistent with permission that I see in Home > Admin > Roles. E.g. for analyst role I see that Task Admin is configured as accessible.

![screenshot from 2017-05-05 12-53-34](https://cloud.githubusercontent.com/assets/2716069/25742963/4a58d62a-3192-11e7-8e03-990290ec6327.png)